### DISCUSSION TITLE
Getting lots of "Unknown command: G17" after updating to v00.04-beta

### DISCUSSION THREAD

**Discussion by DougJoseph** [xblax/flashforge_ad5m_klipper_mod#107] [Q&A] [answered]
Ever since updating to v00.04-beta, I get lots of "Unknown command: G17."

It seems to have to do with setting a plane (for handling arcs?).

I was not seeing this while on the v00.03 beta.

I did not change my slicer config. Any ideas?

**Comment by Tiwatz** [comment, marked as answer]
Set z-hop to "normal" in Orca slicer.

**Comment by DougJoseph** [comment]
Thanks. Have made the change and will test when the current print job finishes!

**Comment by DougJoseph** [comment]
PS: To do this in OrcaSlicer, for anyone reading who does not know:

Click on the printer name in the upper left corner, then click the "Extruder" and look for the "Retraction" section.

**Comment by consp** [comment]
> Ever since updating to v00.04-beta, I get lots of "Unknown command: G17."

That was probably due to the alpha containing newer klipper code which already allows G17 (I mixed two versions accidentally), since that code is not (yet) included it will give you those "errors". It's not really an issue though, just annoying.

**Reply by DougJoseph** [reply]
Got it. Thanks!
